### ISSUE TITLE
Velocity 

### ISSUE THREAD

**Issue by sard0r** [PIC4SeR/AcT#1]
Good job @simoneangarano and thank you. 

I just wanted to ask how do you get velocity features from the openpose pose estimation? do you subtract x2-y2 to get the velocity?

And do you have an inference.py possible or pretrained model for the purpose of testing. 

Thank you very much.

**Comment by simoneangarano** [comment]
Hi @sard0r,
Yes, the velocities are computed as the difference between coordinates at successive time instants.
To test our model you can use main.py, as described in the readme.

**Comment by sard0r** [comment]
@simoneangarano Thank you for your answer. 

I tried to use your model to train and test on my pose data from openpose. So, it is overfitting to much. I am attaching the picture here. 5 clasess. data size is about 1500 videos on train and 300 on test. 
![Screenshot from 2022-06-03 03-31-34](https://user-images.githubusercontent.com/61651742/171701811-47dd2bb0-0255-4bc3-8f67-d03f12ee11be.png)

I am using the base model. What do you suggest me to do. 

Thank you.

**Comment by simoneangarano** [comment]
Hi, 
I don't think it's overfitting much, actually. A 2% gap between training and validation is usually ok!
Am I missing something?
Also, maybe I'd try a smaller learning rate as your training loss is oscillating a bit.
I hope this helps!
